### PR TITLE
doc: complete documentation for ceph fs administration commands

### PR DESCRIPTION
**Pull request by batrick**
Fixes: http://tracker.ceph.com/issues/24580

Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/mon/MonCommands.h [hunk 1]
```diff
@@ -282,7 +282,7 @@ COMMAND("versions",
  * MDS commands (MDSMonitor.cc)
  */
 
-COMMAND("mds stat", "show MDS status", "mds", "r")
+COMMAND("mds stat", "show MDS status", "mds", "r", FLAG(HIDDEN))
```
Comment (liewegas) [inline src/mon/MonCommands.h]: COMMAND_WITH_FLAG

File: src/mon/MonCommands.h [hunk 2]
```diff
@@ -328,7 +328,7 @@ COMMAND_WITH_FLAG("mds set " \
 COMMAND("mds set_state " \
```
Comment (liewegas) [inline src/mon/MonCommands.h]: _WITH_FLAG

**Comment by batrick** [comment]
Fixed, thanks!

File: doc/cephfs/administration.rst [hunk 3]
```diff
@@ -14,34 +14,62 @@ creation of multiple filesystems use ``ceph fs flag set enable_multiple true``.
 
     fs new <filesystem name> <metadata pool name> <data pool name>
 
+This command creates a new file system. The file system name and metadata pool
+name are self-explanatory. The specified data pool is the default data pool and
+cannot be changed once set.
```
[the remark this answers is not in the document]
Comment (batrick) [reply doc/cephfs/administration.rst]: Ya, fixed!

**Pull request merged**